pkg/api/v1/client/proxy.go

```diff
@@ -157,6 +158,14 @@ func (c ProxyScriptsAPI) ExecuteScript(id, namespace, executionName string, exec
 	return c.getExecutionFromResponse(resp)
 }
 
+func (c ProxyScriptsAPI) Logs(id string) (logs chan output.Output, err error) {
```

Comment:
Will this be implemented in the future? should we add a TODO tag or something like that.

Reply:
DONE!